drawer.html

```diff
@@ -0,0 +1,53 @@
+<div class="o-header__drawer" id="o-header-drawer" role="navigation" aria-label="Drawer menu" data-o-header-drawer data-o-header-drawer--no-js>
+	<div class="o-header__drawer-inner">
+
+		<div class="o-header__drawer-tools">
+			<a class="o-header__drawer-tools-logo" href="/">
+				<span class="o-header__visually-hidden">Financial Times</span>
+			</a>
+			<button type="button" class="o-header__drawer-tools-close" aria-controls="o-header-drawer" title="Close drawer menu">
```

Comment:
Given that this button will only control something when JavaScript is working, we should probably set `aria-controls` using JavaScript. That is, unless this button is hidden when JavaScript is not available.

Reply:
aha, well the drawer does kind of work w/o JS. It gets put to the bottom of the page